app/Controllers/BaseController.php

```diff
@@ -37,6 +37,13 @@ abstract class BaseController extends Controller
      */
     protected $helpers = [];
 
+     /**
+     * Be sure to declare properties for anything you assign.
+     * The creation of dynamic property is deprecated in PHP 8.2.
+     * 
+     * protected $session; 
+     */
```

Comment:
Thank you for clean up.

```suggestion
    /**
     * Be sure to declare properties for anything you assign.
     * The creation of dynamic property is deprecated in PHP 8.2.
     */
    // protected $session;
```